packages/backend/src/tokens/authStatus.ts

```diff
@@ -30,17 +32,22 @@ export type SignedOutState = {
   publishableKey: string;
   isSignedIn: false;
   isInterstitial: false;
+  isUnknown: false;
   toAuth: () => SignedOutAuthObject;
 };
 
 export type InterstitialState = Omit<SignedOutState, 'isInterstitial' | 'status' | 'toAuth'> & {
   status: AuthStatus.Interstitial;
-  isSignedIn: false;
   isInterstitial: true;
-  reason: AuthReason;
   toAuth: () => null;
 };
 
+export type UnknownState = Omit<InterstitialState, 'status' | 'isInterstitial' | 'isUnknown'> & {
```

Comment:
Do you think it'd be a good idea to include a short comment about what this case is supposed to do? I'm talking about the "prevent user code from running" as discussed offline. However, I'm not sure whether it'd be better as a comment,  a separate doc or a combination of both.

Reply:
Let's add it in the docs (as discussed offline).